### PR TITLE
fix(standalone): Make sure to verify the depends and provides

### PR DESCRIPTION
**Pull request by oleorhagen**
Simple fix, making sure that we verify the depends and provides in standalone mode as well. Which was simply forgotten.

Ticket: MEN-6929
Changelog: None

### CODE REVIEW

**Review by lluiscampos — APPROVED** [review]
We should have a test for this, either here or in the acceptance tests.

Caching it in an integration test of an independent repository is too late IMO.

Otherwise, nicely solved!

**Review by kacf — APPROVED** [review]
Nice tests!

File: src/mender-update/cli/cli_test.cpp [hunk 1]
```diff
@@ -463,6 +475,92 @@ artifact_name=test
...
+
+		mtesting::RedirectStreamOutputs output;
+		int exit_status = cli::Main(
+			args, [&tmpdir](context::MenderContext &ctx) { SetTestDir(tmpdir.Path(), ctx); });
+		EXPECT_EQ(exit_status, 1) << exit_status;
+
+		EXPECT_EQ(output.GetCout(), R"(Installing artifact...
+Installation failed. System not modified.
+)");
+		EXPECT_EQ(output.GetCerr(), "");
```
Comment (kacf) [inline src/mender-update/cli/cli_test.cpp:560]: No logging at all? Isn't that a bit suspicious?

Comment (oleorhagen) [reply src/mender-update/cli/cli_test.cpp:560]: Yes. But it is the same for all the tests.

I did a closer look. And the only logging on the path, which is not showing is from the artifact parser. And the reason it is not showing is that the log level is lower than `Info`. So this looks right to me :)

**Pull request merged**